stylesheets/_component.timeline.scss

```diff
@@ -0,0 +1,585 @@
+            border-color: rgba(194, 225, 245, 0);
+            border-right-color: $timeline-content-border;
+            border-width: 8px;
+            margin-top: -8px;
+        }
+    }
+
+    &__title {
+        color: #303e49;
+        display: inline-block;
+        margin: 0 0 0 26px;
+
+        @include respond-min($screen-phone) {
+            margin: 0 0 0 32px;
+        }
+
+        @include respond-min($screen-tablet) {
+            line-height: $line-height-base;
+            margin: 0;
+            width: 60%;
+        }
+    }
+
+    &__note-title {
+        margin: 1em 0;
+    }
+
+    &__text {
+        @include word-wrap();
+
+        margin: .5em 0;
+
+        @include respond-min($screen-tablet) {
+            margin: 1em 0;
+        }
+
+        a {
+            word-break: break-all;
+        }
+
+        .avatar {
+            background-color: color(white);
+            vertical-align: bottom;
+        }
+    }
+
+    &__small {
+        color: color(gray, dark);
+        display: inline-block;
+        font-size: .7em;
+
+        .label {
+            font-size: 100%;
+        }
+    }
+
+    &__date {
+        display: block;
+        font-size: 14px;
+        margin-top: $padding-base;
+
+        @include respond-min($screen-tablet) {
+            display: inline-block;
+            float: right;
+            margin-top: 0;
+            text-align: right;
+            width: 40%;
+        }
+    }
+
+    &__attachments {
+        margin: 1em 0;
+
+        li {
+            display: inline-block;
+            margin-right: .5em;
+            width: 150px;
+        }
+    }
+
+    &__thumbnail-link {
+        border-radius: 4px;
+        cursor: pointer;
+        display: inline-block;
+        max-height: 250px;
+        max-width: 100%;
+        overflow: hidden;
+        position: relative;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__thumbnail {
+        max-height: 250px;
+        max-width: 100%;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__from {
+        display: inline-block;
+        height: 0;
+        opacity: 0;
+        overflow: hidden;
+        width: 0;
+    }
+
+    &__from--show {
+        height: auto;
+        opacity: 1;
+        overflow: visible;
+        padding: 0 3px;
+        width: auto;
+
+        .diff-multi-line & {
+            display: block;
+            padding: 5px 0;
+        }
+    }
+
+    &__from-label {
+        transition: background-color .5s linear, color .5s ease;
+    }
+
+    &__from-label--show,
+    &__removed {
+        background: #f6b6b4;
+        border-radius: 3px;
+        color: #8c291c;
+        padding: 1px 2px;
+        text-decoration: line-through;
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__to {
+        background: #abdfab;
+        border-radius: 3px;
+        color: #3e6441;
+        padding: 1px 2px;
+    }
```

Comment:
lots more hardcoded colours here 👆 , could become a bit of a pain when we need to make sweeping brand or contrast changes